app/helpers/email_helper.rb

```diff
@@ -28,4 +27,12 @@ def whitehall_base_email_sign_up_url
   def equivalent_whitehall_url
     root + request.fullpath
   end
+
+  def is_accordion?(presented_taxon)
+    presented_taxon.rendering_type == TaxonPresenter::ACCORDION
+  end
+
+  def is_handled_by_whitehall?(presented_taxon)
+    presented_taxon.base_path.starts_with?(WHITEHALL_BOUND_BASE_PATH) if presented_taxon.base_path
```

Comment:
The `Taxon` class has the `world_related?` method, we could use that since it appears to do the same